portfolio/src/main/webapp/css/custom.css

```diff
@@ -423,7 +423,7 @@ body {
 
 .mb-5,
 .my-5 {
-  margin-bottom: 0.5rem;
+  margin-bottom: 0.5rem !important;
```

Comment:
nit: i believe tong mentioned this in another pr but try to stay away from the use of !important

Reply:
He did, but i found that removing it doesn't allow the custom.css file to overwrite the bootstrap css.

An alternative is deleting the important flag in the styles.css file, but I was refraining from editing things in that file.